windows/internal/nightly_defaults.bat

```diff
@@ -148,7 +148,7 @@ if "%PYTORCH_BUILD_VERSION%" == "" set PYTORCH_BUILD_VERSION=1.5.0.dev%NIGHTLIES
 
 if "%~1" == "Wheels" (
     if "%BUILD_PYTHONLESS%" == "" (
-        if not "%CUDA_VERSION%" == "102" (
+        if not "%CUDA_VERSION%" == "110" (
```

Comment:
So we use cuda 11 as default, cc @seemethere ?

Reply:
I reverted this file in https://github.com/pytorch/builder/pull/463
We could do the change late if we decide to.